src/Veldrid.Utilities/MtlParser.cs

```diff
@@ -91,78 +91,99 @@ public void Process(string line)
                 switch (pieces[0].ToLowerInvariant().Trim())
                 {
                     case "newmtl":
-                        ExpectExactly(pieces, 1, "newmtl");
+                        ExpectAtLeast(pieces, 1, "newmtl");
                         FinalizeCurrentMaterial();
-                        _currentDefinition = new MaterialDefinition(pieces[1]);
+                        if (!DictMatDef.ContainsKey(pieces[1]))
+                        {
+                            DictMatDef.Add(pieces[1], pieces[1]);
+                            _currentDefinition = new MaterialDefinition(pieces[1]);
```

Comment:
The name of the MaterialDefinition should be a concatenation of all pieces in the line, not just the first. Your example should give `"mat_lionstatue copy"`, and not just `"mat_lionstatue"`. This will matter if the MTL file, for whatever reason, included both `newmtl thing copy` and `newmtl thing copy 2`.